### PR TITLE
Improve feature flags ui

### PR DESCRIPTION
**Pull request by esoergel**
@dimagi/product @biyeun 

I made a few minor changes to the feature flags UI to address some of the concerns raised at the summit.

feature-flags

### CODE REVIEW

File: corehq/toggles.py [hunk 1]
```diff
@@ -5,13 +5,43 @@
...
-TAG_PREVIEW = Tag(name='Preview', css_class='default')
-TAG_UNKNOWN = Tag(name='Unknown', css_class='default')
+Tag = namedtuple('Tag', 'name css_class description')
+TAG_ONE_OFF = Tag(
+    name='One-Off',
+    css_class='danger',
+    description="This feature flag was created for one specific use-case. "
+    "Please don't enable it for your project without first talking to the tech "
+    "team. This is not fully supported and may break other features.",
+)
```
Comment (esoergel) [inline corehq/toggles.py]: @dimagi/product This is the main addition of this PR.  We can now add descriptions of what each tag means and the appropriate description will appear on each feature flag page.  This should hopefully reduce any ambiguity around what feature flags people should/shouldn't use.

If we decide to merge this, I'd appreciate it if someone from product took a pass at these descriptions.

**Comment by amsagoff** [comment]
very cool!

File: corehq/toggles.py [hunk 2]
```diff
@@ -5,13 +5,43 @@
...
+    css_class='success',
+    description="This is a core-product feature that you should feel free to "
+    "use.  We've feature-flagged it probably because it is an advanced "
+    "workflow we'd like more control over.",
+)
+TAG_PREVIEW = Tag(
+    name='Preview',
+    css_class='default',
+    description="",  # I'm not sure...
+)
```
Comment (esoergel) [inline corehq/toggles.py]: What does it mean for a feature to be flagged as "Preview", exactly?  Looks like currently this is only enabled for `Access to map-based demo reports`.  I propose we drop it.  Can we also drop the "Unknown" tag?

File: corehq/toggles.py [hunk 3]
```diff
@@ -557,13 +587,20 @@ def toggle_values_by_name(username=None, domain=None):
...
 )
 
 FORM_SUBMISSION_BLACKLIST = StaticToggle(
     'FORM_SUBMISSION_BLACKLIST',
     ("Blacklist form submissions from a domain that spams us"),
     TAG_EXPERIMENTAL,
     namespaces=[NAMESPACE_DOMAIN],
+    description="This is a temporary solution to an unusually high volume of "
+    "form submissions from a domain.  We have some projects that automatically "
+    "send forms. If that ever causes problems, we can use this to cut them off.",
```
Comment (esoergel) [inline corehq/toggles.py]: We've actually had the ability to add a verbose description of a feature flag for a while.  Perhaps someone from @dimagi/product would be interested in taking a pass at existing feature flags and adding descriptions?

**Comment by biyeun** [comment]
@dimagi/product @esoergel this looks good implementation-wise to me. Do you have any comments on the description copy? You can view it in the line-by-line https://github.com/dimagi/commcare-hq/pull/13569/files

**Comment by amsagoff** [comment]
will do a pass!

**Comment by amsagoff** [comment]
This turned into a larger discussion - I think you can move forward without
blocking on the descriptions for now

**Comment by esoergel** [comment]
@amsagoff Sounds good.  It'll also be pretty easy to make modifications in the future.
@sravfeyn @kaapstorm this is g2m on the product side.

**Comment by kaapstorm** [comment]
This is great! And I support your suggestion of dropping PREVIEW and UNKNOWN. They don't seem to add value.

**Pull request merged**